### PR TITLE
Add a goodie for showing the country corresponding to a ccTLD

### PR DESCRIPTION
**Pull request by ehsan**
This implements the idea here: https://duck.co/ideas/idea/1606/country-s-domain

### CODE REVIEW

**Comment by javathunderman** [comment]
@moollaza I'd love to see this go live- I can take this up, unless this has already been implemented elsewhere.

**Comment by mintsoft** [comment]
@javathunderman I can't find any overlapping IA's that are live so I'd say go for it!

**Comment by moollaza** [comment]
:+1:

**Comment by javathunderman** [comment]
Apparently rediscovered this while browsing through Abandoned labeled PRs...will get started. :)

File: lib/DDG/Goodie/CCTLD.pm [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package DDG::Goodie::CCTLD;
+
+use DDG::Goodie;
+
+my %triggers = share('cctld.txt')->slurp;
```
Comment (javathunderman) [inline lib/DDG/Goodie/CCTLD.pm]: @jagtalon Browsing over this list of ccTLDs looks accurate- will implement what you suggested.

File: share/goodie/cctld/README [hunk 2]
```diff
@@ -0,0 +1 @@
+The source of this data is http://mbrownnyc.wordpress.com/misc/iso-3166-cctld-csv/.
```
Comment (javathunderman) [inline share/goodie/cctld/README]: @jagtalon Browsing over this list of ccTLDs looks accurate- will implement what you suggested.

**Comment by javathunderman** [comment]
@moollaza as @MrChrisW pointed out in #1120, most of these ccTLDs are handled by the Wiki IA...Therefore I think we can get rid of the Abandoned label.

**Pull request closed without merging**